packages/cms-cli/lib/createFunctionPrompt.js

```diff
@@ -0,0 +1,66 @@
+const inquirer = require('inquirer');
+
+const { STRING_WITH_NO_SPACES_REGEX } = require('./regex');
+
+const FUNCTIONS_FOLDER_PROMPT = {
+  name: 'functionsFolder',
+  message: 'Name of the folder where your function will be created',
+  validate(val) {
+    if (typeof val !== 'string') {
+      return 'You entered an invalid name. Please try again.';
+    } else if (!val.length) {
+      return 'The name may not be blank. Please try again.';
+    } else if (!STRING_WITH_NO_SPACES_REGEX.test(val)) {
+      return 'The name may not contain spaces. Please try again.';
+    }
+    return true;
+  },
+};
+const ENDPOINT_PATH_PROMPT = {
+  name: 'endpointPath',
+  message: 'Path portion of the URL created for the function',
+  validate(val) {
+    if (typeof val !== 'string') {
+      return 'You entered an invalid name. Please try again.';
+    } else if (!val.length) {
+      return 'The name may not be blank. Please try again.';
+    } else if (!STRING_WITH_NO_SPACES_REGEX.test(val)) {
+      return 'The name may not contain spaces. Please try again.';
+    }
+    return true;
+  },
+};
+const ENDPOINT_METHOD_PROMPT = {
+  type: 'list',
```

Comment:
Do we want to use `rawList` here? We had feedback from users of `list` that they would prefer to be able to select with both arrows as well as numbers.